### PR TITLE
Get the JSON metadata and query for collections

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,6 +85,13 @@ object_client.metadata.dublin_core
 
 # Get the public descriptive XML representation
 object_client.metadata.descriptive
+
+# Return the Cocina metadata
+object_client.find
```
Comment (mjgiarlo) [inline README.md]: I don't love this name. Open to alternatives? Maybe `#get`?

Comment (jcoyne) [reply README.md]: I'm trying to stick close to valkyrie: https://github.com/samvera/valkyrie/blob/master/lib/valkyrie/persistence/memory/query_service.rb#L18-L28

File: README.md [hunk 2]
```diff
@@ -85,6 +85,13 @@ object_client.metadata.dublin_core
...
+
+# Query for an objects collections
+object_client.queries.collections
```
Comment (justinlittman) [inline README.md]: Not a show stopper for me, but don't love `queries`. Everything is a query. Or nothing is a query.

File: README.md (hunk 1 above)
Comment (mjgiarlo) [reply README.md]: ok, sold.

File: README.md (hunk 2 above)
Comment (jcoyne) [reply README.md]: Except the things that are writes.  I can get behind moving find to `Queries` if you like that.

Comment (justinlittman) [reply README.md]: I don't feel strongly; defer to your judgment.

Comment (jcoyne) [reply README.md]: I've relented and done it the jlitt way.

**Pull request merged**